docusaurus/docs-guides/hsm/softhsm.mdx

```diff
@@ -1,1301 +1,1301 @@
-import Tabs from '@theme/Tabs';

```

Comment:
why are there so many changes on this one file?

Reply:
The _real_ differences are highlighted but everything is shaded so they're really inconvenient to see.

Reply:
```bash
git diff --ignore-cr-at-eol main -- docusaurus/docs-guides/hsm/softhsm.mdx
```

Reply:
Can you please redo the PR with just the link changes and not everything else?

Reply:
Yep. https://github.com/openziti/ziti-doc/pull/279